docs/source/general/api.rst

```diff
@@ -2,6 +2,24 @@
 API Documentation
 =================
 
+django-freeradius provides an API for performing all the get and post
+actions. Only authorized users with a token will able to get and post to
+the API. The API token can be configured in the settings which is mandatory.
+
+``DJANGO_FREERADIUS_API_TOKEN``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Set this variable to the secret API token of your instance.
+Users can send this token as a bearer token or also as a query string.
+FreeRADIUS must be configured to send a bearer token as mentioned in the
+`docs <freeradius.html>`_.
+
+Accessing the API
+-----------------
+
+To access the API you will need to send the token as a querystring or a
+bearer token in the Authorization header.
```

Comment:
Please mention the two options in a list, put the bearer token option first and write `(recommended)` besides it.

Then please provide 2 examples with `curl`, one with the bearer token header and one with the querystring.